library/core/src/iter/sources.rs

```diff
@@ -5,9 +5,8 @@ use super::{FusedIterator, TrustedLen};
 
 /// An iterator that repeats an element endlessly.
 ///
-/// This `struct` is created by the [`repeat`] function. See its documentation for more.
-///
-/// [`repeat`]: fn.repeat.html
+/// This `struct` is created by [`repeat`].
```

Comment:
```suggestion
/// This `struct` is created by [`repeat()`].
```

Reply:
I don't know if this is good since for other pages I didn't add `()`, I personally think it is unnecessary since it is pretty obvious `repeat` means a function. Is there any other possibility that `repeat` could be other than a function?

Reply:
`repeat` is not very clear. It's idiomatically referred to as `iter::repeat`, so saying just 'repeat' doesn't immediately bring it to mind.

Reply:
I made it to use `iter::repeat` but not `iter::repeat()`. I do think both of your suggestion is good but probably not `()`, I don't have a good viewpoint but I think not having it is better.

Reply:
Ok, I don't feel very strongly about `()` as long as it has the module prefix.